.travis.yml

```diff
@@ -13,16 +13,33 @@ addons:
             - liblapack-dev
 
 cache:
-    apt: true
-    pip: true
-    ccache: true
+    apt: true   # does not work for public repos
+    directories:
+        - $HOME/.cache/pip
+        - $HOME/.ccache
+        - $HOME/nltk_data
+        - $TRAVIS_BUILD_DIR/pyqt
+
+before_cache:   # prevent logs from caching
+    - rm -f $HOME/.cache/pip/log/debug.log
```

Comment:
One achieves the same with `cache: pip: false`, or by installing our sw with `python setup.py install` instead of pip. Try the latter.